SPECS/squashfs-tools/squashfs-tools.spec

```diff
@@ -1,41 +1,15 @@
 Summary: Utility for the creation of squashfs filesystems
 Name: squashfs-tools
-Version: 4.3
-Release: 26%{?dist}
+Version: 4.4
+Release: 1%{?dist}
 License: GPLv2+
-URL: http://squashfs.sourceforge.net/
-Source0: http://downloads.sourceforge.net/squashfs/squashfs%{version}.tar.gz
-# manpages from http://ftp.debian.org/debian/pool/main/s/squashfs-tools/squashfs-tools_4.2+20121212-1.debian.tar.xz
-# The man pages have been modified for 4.3 for Fedora.
+URL: https://github.com/plougher/squashfs-tools
+# source0: https://github.com/plougher/squashfs-tools/archive/refs/tags/4.4.tar.gz
+Source0: %{name}-%{version}.tar.gz
 Source1: mksquashfs.1
```

Comment:
Just confirming: Do these documentation sources need to be updated for the new version?

Reply:
the old URL http://squashfs.sourceforge.net/ now only refers to the new one and release mechanism has moved from sourceforge to github as well